stock_prices/stock_prices.py

```diff
@@ -3,11 +3,37 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
-
+  #Receive a series of prices
+  #use test example [1050, 270, 1540, 3800, 2]
+  
+  #Find largest difference -> Negative indicate a loss
+  
+  # Math goes like this by iterating through
+  # 1050 -> Buy, no profit made
+  # 270 -> buy, no profit 270 - 1050 =  -780
+  # 1540 subtract with previous numbers -> 1540-270 or 1540-1050 -> profit is made on both, records these
+  # 3800 subtract with previous numbers, largest profit so far
+  # 2 is last number and isn't profitable
+  
+  # Use the hints
+  
+  #Max profit math is subtracting some price by another price that comes before it
+  
+    current_min_price_so_far = prices[0]#start with first number
+    max_profit_so_far = 0 #profit will zero at start
+    for i in range(len(prices)):#loop through given prices
```

Comment:
Great work! Remember Python has `max()` and `min()` built in functions that may have helped you here